src/pages/Compare/index.tsx

```diff
@@ -31,7 +31,7 @@ type PropsFromRouter = {
 
 type PropsFromState = {
   addonId: number;
-  compareInfo: CompareInfo | null | void;
+  compareInfo: CompareInfo | null | undefined;
```

Comment:
I thought `void` was preferred, no? If it's trying to match another type, maybe change that one?

Reply:
ahm, this is not needed actually